packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -107,8 +127,13 @@ const innerComponent = forwardRef<SquiggleViewerHandle, SquiggleViewerProps>(
         partialPlaygroundSettings={partialPlaygroundSettings}
         localSettingsEnabled={localSettingsEnabled}
         editor={editor}
+        beginWithVariablesCollapsed={resultItem !== undefined && resultItem.ok}
       >
-        <SquiggleViewerOuter result={result} ref={ref} />
+        <SquiggleViewerOuter
+          resultVariables={resultVariables}
+          resultItem={resultItem}
```

Comment:
Given that they share the same error, I was thinking it could make sense to combine these into one type.

Reply:
I'm pretty sure this (and `DynamicSquiggleViewer`) should just pass `SquiggleOutput["output"]` instead of two separate props. `lib/utility.ts` functions could be removed/moved to `SquiggleViewerOuter` above.